packages/safety-suite-api/src/api/models/types.ts

```diff
@@ -115,23 +115,281 @@ export type OrderBy = {
 };
 
 export type ModelListOptions = {
+  /**
+   * Maximum number of models to return.
+   */
   limit?: number;
+  /**
+   * Options for sorting the returned models.
+   */
   orderBy?: OrderBy;
+  /**
+   * Index in the array of returned models.
```

Comment:
Just to be clear - this one omits the first `start` models from the result set, so the first model of response would indeed have a `start` index of the initial result set.